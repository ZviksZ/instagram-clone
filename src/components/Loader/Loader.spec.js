import React from "react";
import {Loader} from './Loader';

const setUp = () => shallow(<Loader />);

describe("Loader component", () => {
   it("should render Loader", () => {
      const component = setUp();
      expect(component).toMatchSnapshot();
   });
   it("should render Loader wrapper", () => {
      const component = setUp();

      const wrapper = component.find('.center-text');
      expect(wrapper.length).toBe(1);
   });
});
