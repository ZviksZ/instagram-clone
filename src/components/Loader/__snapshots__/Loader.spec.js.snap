// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render Loader 1`] = `
<div
  className="center-text mt50"
>
  <WithStyles(ForwardRef(CircularProgress)) />
</div>
`;
